spam/spam_tutorial.py

```diff
@@ -16,7 +18,7 @@
 #
 # 2. **Writing Labeling Functions**: We write Python programs that take as input a data point and assign labels (or abstain) using heuristics, pattern matching, and third-party models.
 #
-# 3. **Combining Weak Labels with the Label Model**: We use the outputs of the labeling functions over the training set as input to the label model, which assings probabilistic labels to the training set.
+# 3. **Combining Labeling Functions with the Label Model**: We use the outputs of the labeling functions over the training set as input to the label model, which assings probabilistic labels to the training set.
```

Comment:
assings -> assigns

Reply:
that is both more correct and less potentially offensive. good call @brahmaneya.